src/server/scripts/Northrend/Ulduar/Ulduar/boss_xt002.cpp

```diff
@@ -91,17 +91,20 @@ enum NPCs
     NPC_PILE_TRIGGER            = 33337,
 };
 
-enum Sounds
+enum XT002Text
 {
-    XT_SOUND_AGGRO              = 15724,
-    XT_SOUND_HEART_OPEN         = 15725,
-    XT_SOUND_HEART_CLOSED       = 15726,
-    XT_SOUND_TANTARUM           = 15727,
-    XT_SOUND_SLAY1              = 15728,
-    XT_SOUND_SLAY2              = 15729,
-    XT_SOUND_ENRAGE             = 15730,
-    XT_SOUND_DEATH              = 15731,
-    XT_SOUND_SUMMON             = 15732,
+    SAY_AGGRO                   = 0,
+    SAY_HEART_OPENED            = 1,
+    SAY_HEART_CLOSED            = 2,
+    SAY_TYMPANIC_TANTRUM        = 3,
+    SAY_SLAY                    = 4,
+    SAY_BERSERK                 = 5,
+    SAY_DEATH                   = 6,
+    SAY_SUMMON                  = 7,
+    SAY_EMOTE_HEART_OPENED      = 8,
+    SAY_EMOTE_HEART_CLOSED      = 9,
+    SAY_EMOTE_TYMPANIC_TANTRUM  = 10,
+    SAY_EMOTE_SCRAPBOT          = 11,
```

Comment:
should just be EMOTE_

Reply:
Done, good to know I guess.

Reply:
https://www.azerothcore.org/wiki/cpp-code-standards#standard-prefixes-for-constants if you want to learn more